test/manual/gbitmap.js

```diff
@@ -200,11 +207,19 @@ describe('GBitmap', function() {
         delete symbols.Resources.defaultProxy;
```

Comment:
This is a misnomer. Instead, should be "returns the url directly when no proxy is set"